docs/sources/shared/visualizations/stack-series-link.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Stack series link
+---
+
```

Comment:
Needs a stem sentence.

Reply:
What do you mean?

Reply:
Introduction, introducing ...

Reply:
In the title?

Or are you talking about before or after the heading? All the content that would be under this heading is in another topic. The heading exists to maintain consistency with the rest of the option entries and to hold the paragraph with the link pointing to the topic where the content lives.

Reply:
After the title.

Reply:
That is not needed for this content snippet. The snippet starts with the heading.

Reply:
Ah - ok. Got it.